Dockerfile.test.rhel8

```diff
@@ -0,0 +1,38 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.3
+
+ENV INSTALL_DIR=/opt/conjur-api-python3
+
+# Copy below repo file to enable installation of gnome-keyring and dbus-x11
+COPY ./CentOS-Linux-AppStream.repo /etc/yum.repos.d/CentOS-Linux-AppStream.repo
+
+RUN yum --disableplugin=subscription-manager -y install -y \
+            bash \
```

Comment:
indentation. I think all the following dependencies should be right below the `\` in the previous line or `install`